class.jetpack-gutenberg.php

```diff
@@ -84,30 +84,42 @@ public static function load_blocks() {
 	}
 
 	/**
-	 * Checks for a given .json file in the `_inc/blocks` folder.
+	 * Return the Gutenberg extensions (blocks and plugins) directory
+	 *
+	 * @return string The Gutenberg extensions directory
+	 */
+	public static function get_blocks_directory() {
+		if ( function_exists( 'wpcom_is_proxied_request' ) && wpcom_is_proxied_request() ) {
```

Comment:
instead of doing this can we just add a filer for this? 
This way we can just filter it on .com and not have to reply on function exiting check